packages/workflows-sdk/src/utils/composer/create-workflow.ts

```diff
@@ -256,5 +263,36 @@ export function createWorkflow<
 
   mainFlow.getName = () => name
 
+  mainFlow.run = ({
+    input,
+  }: {
+    input: TData
+  }): ReturnType<StepFunction<TData, TResult>> => {
+    // TODO: Async sub workflow is not supported yet
+    // Info: Once the export workflow can fire the execution through the engine if loaded, the async workflow can be executed,
+    // the step would inherit the async configuration and subscribe to the onFinish event of the sub worklow and mark itself as success or failure
+    return createStep(
```

Comment:
ayy very cool! 🎉 

Just out of curiosity, do you think it would be possible export this directly?

Something like:

```
export const workflow = createWorkflow(
  workflowId,
  (input => {
    SomeStep(input)
    
    return WorkflowStep(differentWorkflow, input)
  })
```

Reply:
I ve slightly change the API to unify them all 👍 wdyt?